tests/ClassNameSuffixFixerTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests;
+
+use Landingi\ClassNameSuffixFixer;
+use PhpCsFixer\Tokenizer\Tokens;
+use PHPUnit\Framework\TestCase;
+
+class ClassNameSuffixFixerTest extends TestCase
+{
+    private ClassNameSuffixFixer $fixer;
+
+    protected function setUp(): void
+    {
+        $this->fixer = new ClassNameSuffixFixer();
+    }
+
+    private function getValidPhpCode(): string
+    {
+        return '
+        <?php
+
+        class ExampleClass
+        {
+        
+        }';
+    }
+
+    public function testDoesNotFixAnything(): void
+    {
+        $tokens = Tokens::fromCode($this->getValidPhpCode());
+        $this->fixer->fix(new \SplFileInfo('test.php'), $tokens);
```

Comment:
Przecież nie ma takiego pliku jak test.php

Reply:
Wiesz co ten plik zbytnio nie ma znaczenia. Fix opiera sie na tych tokenach - to jest ten caly kod, ktory sprawdzamy. Rownie dobrze moge dac tam empty stringa.